tapiriik/services/Hexoskin/hexoskin.py

```diff
@@ -0,0 +1,256 @@
+from tapiriik.settings import WEB_ROOT, HEXOSKIN_CLIENT_SECRET, HEXOSKIN_CLIENT_ID
+from tapiriik.services.service_base import ServiceAuthenticationType, ServiceBase
+from tapiriik.services.service_record import ServiceRecord
+from tapiriik.services.interchange import UploadedActivity, ActivityType, ActivityStatistic, ActivityStatisticUnit
+from tapiriik.services.api import APIException, UserException, UserExceptionType, APIExcludeActivity
+from tapiriik.services.tcx import TCXIO
+
+from django.core.urlresolvers import reverse
+from datetime import datetime, timedelta
+from urllib.parse import urlencode
+import requests
+import logging
+import pytz
+import time
+
+logger = logging.getLogger(__name__)
+serverRoot = "https://api.hexoskin.com/"

```

Comment:
The serverRoot shouldn't change, I'd rather hardcode it in the url strings in the code.